app/clients/sms/aws_sns.py

```diff
@@ -23,7 +23,7 @@ def init_app(self, aws_region, statsd_client, logger, *args, **kwargs):
     def get_name(self):
         return self.name
 
-    def send_sms(self, to, content, reference, multi=True, sender=None):
+    def send_sms(self, to, content, reference, multi=True, sender=None, **kwargs):
```

Comment:
> ditto code smell

This is necessary because each provider has differing methods of sending sms, whereas the named args are usually required. This is completely unrelated to what is in `letters_pdf_tasks.py`

Reply:
Then perhaps my comment on letters_pdf_tasks.py belongs here.  Each provider having differing methods of sending sms is exactly why I think OOP and polymorphism is a bad implementation.